index.html

```diff
@@ -9,6 +9,7 @@
 </head>
 <body>
     <div id="main">
+
```

Comment:
1 - Please use Semantic Tags for HTML. For example:

    Instead of using id='main', You can use tag in your code. But, don't forget to use other semantic tags like section and footer. It will be good If your div tags are inside the semantic tags.